docs/guides/running-puppeteer-in-extensions.md

```diff
@@ -0,0 +1,69 @@
+# Running Puppeteer in Chrome extensions
```

Comment:
@oliverdunk PTAL

Reply:
Thanks! Generally looks great, I left a couple of comments.